assets/section-image-banner.css

```
@@ -181,12 +184,13 @@
     --color-foreground: 255, 255, 255;
     --color-button: 255, 255, 255;
     --color-button-text: 0, 0, 0;
+    max-width: 89.5rem;
```

```suggestion
    max-width: 89rem;
```
Cleaning up some values, the 0.5 didn't make a difference here.